polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -536,17 +536,23 @@ pub mod pallet {
 		/// The unlock operation cannot succeed because there are still consumers of the lock.
 		InUse,
 		/// Invalid asset, reserve chain could not be determined for it.
+		#[codec(index = 21)]
 		InvalidAssetUnknownReserve,
 		/// Invalid asset, do not support remote asset reserves with different fees reserves.
+		#[codec(index = 22)]
 		InvalidAssetUnsupportedReserve,
 		/// Too many assets with different reserve locations have been attempted for transfer.
+		#[codec(index = 23)]
 		TooManyReserves,
 		/// Local XCM execution incomplete.
+		#[codec(index = 24)]
 		LocalExecutionIncomplete,
 		/// Could not decode XCM.
+		#[codec(index = 25)]
 		UnableToDecode,
 		/// XCM encoded length is too large.
 		/// Returned when an XCM encoded length is larger than `MaxXcmEncodedSize`.
+		#[codec(index = 26)]
```

Comment:
Now what i meant

Reply:
Oh I saw your suggestion. Should I revert and follow your aproach?

Reply:
no revert required, you could do a followup PR

Reply:
Gotcha thanks @acatangiu @ggwpez